File: gherkin/java/src/main/java/gherkin/pickles/PickleRow.java
package gherkin.pickles;

import java.util.List;

import static java.util.Collections.unmodifiableList;

public class PickleRow {
    private final List<PickleCell> cells;

    public PickleRow(List<PickleCell> cells) {
        this.cells = unmodifiableList(cells);
    }

    public List<PickleCell> getCells() {
        return cells;
    }

}

File: gherkin/dotnet/Gherkin/Pickles/PickleRow.cs
using System.Collections.Generic;
using System.Runtime.Serialization;

namespace Gherkin.Pickles
{
    public class PickleRow
    {
        [DataMember(Name = "cells")]
        public IEnumerable<PickleCell> Cells { get; private set; }

        public PickleRow()
        {
            
        }
        
        public PickleRow(IEnumerable<PickleCell> cells)
        {
            Cells = cells;
        }
    }
}
